forest/shared/src/cli/mod.rs

```diff
@@ -87,6 +89,9 @@ pub struct CliOpts {
     /// Daemonize Forest process
     #[structopt(long)]
     pub detach: bool,
+    /// Enable or disable colored logging in `stdout`
+    #[structopt(long)]
+    pub color: Option<LoggingColor>,
```

Comment:
It may be a bit easier and more expressive without `Option` and a default being `auto` (which would be nicely printed even in the `--help`)

Reply:
Sure, this also allows me to clean up that helper function